### ISSUE TITLE
feature request: sshx-server embed static files and start ssl server

### ISSUE THREAD

**Issue by DongHuangT1** [ekzhang/sshx#59]
Since the current self-built service steps are, first the front-end build static files placed into the sshx-server working directory, and then start an https server forwarding to the sshx service listener, so I recommend adding parameters to specify the certificate for ssl listening to start, and the front-end files embedded in the program, eliminating the need to build the front-end and traffic forwarding of these steps.

**Comment by ekzhang** [comment]
Yeah, can add those options to take the TLS chain and keypair as files. Focus right now isn't on self-hosting though. This is not supported, and there are no instructions for you to do this — I'm not sure where you saw that.

Embedding static files into the binary will not be supported.
